microsoft/botframework-solutions
# [DEPENDS ON PR #564][Enterprise Bot Template][TS] Fix TSLint Warnings in individuals files of src folder

## Description

- Fix the warnings that indicates **TSLint** for readability, maintainability, and functionality.

## Related Issue
Fix [#580 ](https://github.com/Microsoft/AI/issues/580)

## Testing Steps

1. Open `AI/templates/Enterprise-Template/src/typescripts`.
2. Run `npm i` (to install tslint dependency).
3. Run `npm run lint` and check there is no warnings in those files.

## Checklist
~- [ ] I have commented my code, particularly in hard-to-understand areas~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
